package.json

```diff
@@ -45,32 +45,32 @@
     "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "17.4.4",
-    "@commitlint/config-conventional": "17.4.4",
-    "@rollup/plugin-typescript": "11.0.0",
-    "@tinyhttp/app": "2.0.29",
+    "@commitlint/cli": "17.7.2",
+    "@commitlint/config-conventional": "17.7.0",
+    "@rollup/plugin-typescript": "11.1.4",
+    "@tinyhttp/app": "2.0.32",
     "@tinyhttp/cookie-parser": "2.0.6",
-    "@types/express-session": "1.17.6",
-    "@types/node": "18.15.3",
-    "@typescript-eslint/eslint-plugin": "5.55.0",
-    "@typescript-eslint/parser": "5.55.0",
-    "c8": "7.13.0",
-    "eslint": "8.36.0",
-    "eslint-config-prettier": "8.7.0",
-    "eslint-plugin-prettier": "4.2.1",
+    "@types/express-session": "1.17.8",
+    "@types/node": "20.8.0",
+    "@typescript-eslint/eslint-plugin": "6.7.3",
+    "@typescript-eslint/parser": "6.7.3",
+    "c8": "8.0.1",
+    "eslint": "8.50.0",
+    "eslint-config-prettier": "9.0.0",
+    "eslint-plugin-prettier": "5.0.0",
     "express-session": "1.17.3",
     "husky": "8.0.3",
-    "milliparsec": "2.2.2",
-    "prettier": "2.8.4",
-    "rollup": "3.19.1",
```

Comment:
btw we can get rid of roll-up completely and compile using TSC

examples:

- https://github.com/tinyhttp/unless
- https://github.com/tinyhttp/logger

Reply:
I want to do that on a separate PR